src/map/utils/trustutils.cpp

```diff
@@ -565,13 +570,34 @@ namespace trustutils
             sJobStat = grade::GetHPScale(grade, baseValueColumn) + (grade::GetHPScale(grade, scaleTo60Column) * (sLvl - 1)) +
                        (grade::GetHPScale(grade, scaleOver30Column) * subLevelOver30) + subLevelOver30 + subLevelOver10;
         }
+        
+        if (mLvl >= 99 && mLvl < 105)
```

Comment:
All of these conditions are exclusive, and should not be in separate statements, but instead if/else if.  Also, after the first check if using that pattern, the lower bound becomes unnecessary.

Reply:
If (mlvl >= base)
{
    Do
}
else if (mlvl >= higher value)
{
    Do
}
Etc?

Reply:
Actually, now that I think about it, invert the conditions.

if level >= upper condition

elseif level >= condition below that

etc

Reply:
This has been restacked upper conditions to lower conditions.